opts/opts.go

```diff
@@ -222,6 +225,9 @@ func ValidateEnv(val string) (string, error) {
 	if len(arr) > 1 {
 		return val, nil
 	}
+	if strings.ContainsAny(val, whiteSpaces) {
```

Comment:
Odd that we only check for whitespace when there's a "=" in 'val'.
Seems like we should check for the validity of the chars of the variable name in all cases by doing this if-statement (as a regex) irrespective of whether there's a "=" in there or not.